examples/vanilla-prosemirror/.env.production

```
@@ -0,0 +1,2 @@
+VITE_YORKIE_API_ADDR='https://api.yorkie.dev'
+VITE_YORKIE_API_KEY='D6sd9E5ehtXBssmeRZ4QQP'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)